packages/x-components/src/x-modules/experience-controls/store/types.ts

```diff
@@ -0,0 +1,52 @@
+import { XStoreModule } from '../../../store/store.types';
+import { ExperienceControlsConfig } from '../config.types';
+
+/**
+ * Experience Controls store state.
+ *
+ * @public
+ */
+export interface ExperienceControlsState {
+  /**
+   * Configuration for the `ExperienceControls` module.
+   */
+  config: ExperienceControlsConfig;
```

Comment:
The `config` parameter in the modules is somewhat reserved for configurations that affect how the module behaves and intended to be set in the installation (in the `XInstaller`). I wouldn't use it for this case.
Instead a field named something like `controls` or similar makes more sense for this case.

Remember that in the state you should also have a place to store the `events` that will be populated by the adapter from the response of the endpoint and emitted.